db/migrations/2023081614350001-change-working-to-userId-data-issues-table.js

```diff
@@ -0,0 +1,80 @@
+const { Op } = require("sequelize");
+const ChainModel = require("../models/chain.model");
+const NetworkModel = require("../models/network.model");
+const IssueModel = require("../models/issue.model");
+const TokenModel = require("../models/tokens.model");
+const RepositoryModel = require("../models/repositories.model");
+const PullRequestModel = require("../models/pullRequest.model");
+const MergeProposalModel = require("../models/mergeproposal");
+const CuratorsModel = require("../models/curator-model");
+const BenefactorModel = require("../models/benefactor.model");
+const DisputeModel = require("../models/dispute-model");
+const UserPaymentsModel = require("../models/user-payments");
+const DeveloperModel = require("../models/developer.model");
+const CommentsModel = require("../models/comments.model");
+const UserModel = require("../models/user");
+
+const { SKIP_MIGRATION_SEED_WORKING_WITH_USER_ID_ISSUE } = process.env;
+
+async function up(queryInterface, Sequelize) {
+  if (SKIP_MIGRATION_SEED_WORKING_WITH_USER_ID_ISSUE === "true") return;
```

Comment:
skipping migrations should only exist when the migration is data-heavy -- this is just replacing some things in the database, we should never give the option to skip as without this migration the app will break